tests/anmykh/test_homework_3.py

```diff
@@ -0,0 +1,21 @@
+def test_list():
+    assert list("abc") == ["a", "b", "c"]
+
+
+def test_empty():
+    assert int() == 0
+    assert bool() == False
+    assert float() == 0.0
+    assert str() == ''
+    assert tuple() == ()
+    assert list() == []
+    assert range(10) == range(0, 10, 1)
+    assert set() is not ()
```

Comment:
incorrect check 
is - identity check
you need to compare values or types

set() != ()


a = ()
type(a) is not set